tsconfig.json

```diff
@@ -47,7 +47,7 @@
       "path": "packages/exporter-trace-otlp-grpc"
     },
     {
-      "path": "packages/exporter-trace-otlp-http/tsconfig.all.json"
```

Comment:
This automatic in place fixing is introduced with https://github.com/open-telemetry/opentelemetry-js/pull/2556 as it is not recognized with [update-ts-references ](https://github.com/open-telemetry/opentelemetry-js/blob/main/package.json#L39). Maybe we should disable automatic update-ts-references or remove tsconfig.all.json to not mess up the workspace on bootstrap (especially to newcomers, this is confusing that the git tracked files are updated on bootstraping).

Reply:
@Flarna would you mind removing these unrelated changes so that we can land this PR? thanks! We can discuss these changes at https://github.com/open-telemetry/opentelemetry-js/pull/2765.

Reply:
done